vhost/src/vhost_user/message.rs

```diff
@@ -948,6 +956,25 @@ enum_value! {
     }
 }
 
+/// Contains UUID to interact with associated virtio shared object.
+#[repr(C)]
+#[derive(Clone, Copy, Default)]
+pub struct VhostUserSharedMsg {
+    /// UUID of the shared object
+    pub uuid: Uuid,
+}
+
+// SAFETY: Safe because VhostUserTransferDeviceState is a POD
```

Comment:
`VhostUserTransferDeviceState` should be replaced

Reply:
Should we also mention that `Uuid` is a POD?

Following the definitions:
```
pub struct Uuid(Bytes);

pub type Bytes = [u8; 16];
```

Reply:
Yeah, sorry. Copy/pasted and then forgot to review the comment before posting.

I guess we can say it is safe because it is a fixed size array?

Reply:
> Yeah, sorry. Copy/pasted and then forgot to review the comment before posting.

Don't worry ;-)
 
> I guess we can say it is safe because it is a fixed size array?

Yep, I think so.